### PR TITLE
Newcomers_Guide.rst: Allow only one newcomer issue

### PR DESCRIPTION
**Pull request by theOriginalMuktaka**
Tells the newcomer to merge only one difficulty/newcomer issue.

Fixes https://github.com/coala/coala/issues/4771

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

### CODE REVIEW

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -23,6 +23,8 @@ following tasks, you will become a developer at coala:
   - merge a ``difficulty/low`` Pull Request
   - review at least a ``difficulty/low`` or higher Pull Request
 
+Note: please merge only *one* ``difficulty/newcomer`` Pull Request
```
Comment (pradeepgangwar) [inline docs/Developers/Newcomers_Guide.rst]: I have a little suggestion. I think P should be capital in **please** . I think anyone else should also have a look. Rest looks good.

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -23,6 +23,8 @@ following tasks, you will become a developer at coala:
   - merge a ``difficulty/low`` Pull Request
   - review at least a ``difficulty/low`` or higher Pull Request
 
+Note: Please merge only *one* ``difficulty/newcomer`` Pull Request
```
Comment (prnvdixit) [inline docs/Developers/Newcomers_Guide.rst]: Actually, this note is intended to be written for contributors (they don't merge PRs :wink:)

Please, rephrase this to something like - 
"Please, don't take up more ``difficulty/newcomer`` issues once you have already solved one. It is recommended to move to higher difficulty issues after that."
:smile:

Comment (theOriginalMuktaka) [reply docs/Developers/Newcomers_Guide.rst]: I used the word 'merge' because it has been used even in ```merge a ``difficulty/newcomer`` Pull Request```.
I'll change my line, please tell me whether I should change the instruction list too.

Comment (prnvdixit) [reply docs/Developers/Newcomers_Guide.rst]: Ahh... I am not sure for that then :sweat_smile:
But anyway, your changes are perfect :smile: :+1:
